.planning/codebase/STRUCTURE.md

```
@@ -1,257 +1,134 @@
 # Directory Structure
 
-## Root Layout
+**Analysis Date:** 2026-08-25
+
+## Top-Level Layout
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)